src/pages/settings/Security/TwoFactorAuth/Steps/EnabledStep.js

```diff
@@ -46,7 +45,7 @@ function EnabledStep() {
                     title={translate('twoFactorAuth.disableTwoFactorAuth')}
                     onConfirm={() => {
                         setIsConfirmModalVisible(false);
-                        setStep(CONST.TWO_FACTOR_AUTH_STEPS.DISABLED);
+                        Navigation.navigate(ROUTES.SETTINGS_2FA.DISABLED, CONST.NAVIGATION.TYPE.FORCED_UP);
```

Comment:
Why do we need `CONST.NAVIGATION.TYPE.FORCED_UP` over default behaviour ?

Reply:
Using `CONST.NAVIGATION.TYPE.FORCED_UP` ensures that when the user gets to the `/success` step and then clicks the back button, they won't be redirected back to the `/enabled` step. They go straight to `/settings/security`, this matches the functionality of the `onBackButtonPress` prop of `HeaderWithBackButton`.